compass/ocean/mesh/lts_regions.py

```diff
@@ -0,0 +1,362 @@
+import math
+import subprocess as sp
+
+import netCDF4 as nc
+import numpy as np
+import xarray
+from shapely.geometry import Point
+from shapely.geometry.polygon import Polygon
+
+from compass.step import Step
+
+
+class LTSRegionsStep(Step):
+    """
+    A step for adding LTS regions to a global MPAS-Ocean mesh
+
+    Attributes
+    ----------
+    cull_mesh_step : CullMeshStep
+        The culled mesh step containing input files to this step
+    """
+    def __init__(self, test_case, cull_mesh_step,
+                 name='lts_regions', subdir='cull_mesh'):
+        """
+        Create a new step
+
+        Parameters
+        ----------
+        test_case : compass.ocean.tests.global_ocean.mesh.Mesh
+            The test case this step belongs to
+
+        cull_mesh_step : cull_mesh_step
+            The culled mesh step containing input files to this step
+
+        name : str, optional
+            the name of the step
+
+        subdir : str, optional
+            the subdirectory for the step.  The default is ``name``
+        """
+        super().__init__(test_case, name=name, subdir=subdir,
+                         cpus_per_task=None, min_cpus_per_task=None)
+        self.cull_mesh_step = cull_mesh_step
+
+    def setup(self):
+        """
+        Set up the test case in the work directory, including downloading any
+        dependencies.
+        """
+        super().setup()
+
+        config = self.cull_mesh_step.config
+        self.cpus_per_task = config.getint('spherical_mesh',
+                                           'cull_mesh_cpus_per_task')
+        self.min_cpus_per_task = config.getint('spherical_mesh',
+                                               'cull_mesh_min_cpus_per_task')
+
+    def run(self):
+        """
+        Run this step of the test case
+        """
+
+        fine_region = get_fine_region(region='east_us_coast')
+
+        label_mesh(fine_region, mesh='culled_mesh.nc',
+                   graph_info='culled_graph.info', num_interface=2)
+
+
+def get_fine_region(region):
```

Comment:
It would be better if these regions were defined in one or more data files rather than hard-coded.  geojson seems like the most natural format but NetCDF would be fine.  The file would be small enough that it could go in the code but I would still put it in the database instead.

Reply:
Done with latest commit.